### DISCUSSION TITLE
Visualize the Nx project Tags in the Workspace Graph

### DISCUSSION THREAD

**Discussion by zackarydev** [nrwl/nx#23411] [Feature Requests]
- [ ] I'd be willing to implement this feature ([contributing guide](https://github.com/nrwl/nx/blob/master/CONTRIBUTING.md))

## Description
It's fairly hard to visualize the complex dependency constraints of tags. I'm sure many projects have tried documenting them in some fashion. If we can have a visual representation of this in the Nx graph we can automate documentation

## Motivation
Documenting these tags is too time consuming. I want a quick way to visualize how they interact.

## Suggested Implementation

Running `nx graph` in the top right, where we select `Project, Tasks`, we could add `Tags`

## Alternate Implementations
`nx graph tags` ?

**Comment by FrozenPandaz** [comment]
I've converted this to a discussion so it's easier to discuss what people would like to see.

**Comment by TheCodeKat** [comment]
I'd find this extremely useful too.

**Comment by hollispotter** [comment]
Yes, please! We're constraining dependency by tags on 4 axes, and I have a hackish Bash line to dump these to CSV (below). Nx Graph is nice, but not practically useful to us without this info. Best would be grouping similar to directories.

Hack in case someone finds it useful:

```sh
npx nx show projects | while read -r i; do npx nx show project "$i" --json | jq -r '.name + "," + .root + ",\"" + (.tags | join(",")) + "\""' ; done > modules.csv
```

**Comment by MillerSvt** [comment]
@FrozenPandaz Hi. If this direction makes sense to the Nx team, I'd be happy to take a shot at implementing it and open a PR, if you're open to this kind of contribution.
